frontend/src/components/MainView/MainView.tsx

```diff
@@ -75,14 +77,20 @@ function Admin(props: LoggedInViewProps) {
         <PrivateRoute path="/users">
           <Users />
         </PrivateRoute>
-        <PrivateRoute exact path="/courses">
+        <PrivateRoute path="/courses">
           <CourseList/>
         </PrivateRoute>
         <PrivateRoute exact path="/coursecreate">
           <CourseCreate/>
         </PrivateRoute>
         <PrivateRoute path="/courses/:id" component={Course}>
         </PrivateRoute>
+        <PrivateRoute path="/sections/:id/edit">
+          <SectionView />
+        </PrivateRoute>
+        <PrivateRoute path="/sections/create">
+          <SectionView />
+        </PrivateRoute>
         <PrivateRoute path="/sections">
           <Sections />
```

Comment:
I think we should add component ManageSections directly in route and delete function Sections(). Function Sections (and others) were created to show how routing works. Do it like below:

<PrivateRoute path="/sections">
          <ManageSections />
 <PrivateRoute/>

Reply:
Right, done 👍